src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/fragments/SelectBook.kt

```diff
@@ -1,27 +1,35 @@
 package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments
 
 import javafx.geometry.Pos
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.viewmodel.ProjectCreationViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.projecthome.ProjectHomeView
 import tornadofx.*
 
-class SelectBook() : View() {
+
+class SelectBook : View() {
     val viewModel: ProjectCreationViewModel by inject()
-    //val root = DataGrid<Project>()
-//    var mappedList: ObservableList<Project>
-    init {
-//        mappedList = viewmodel.projectsProperty.value.map {
-//            add
-//        }
-    }
 
     override val root =
-            datagrid(listOf("string")) {
+            datagrid(viewModel.bookList) {
+                bindSelected(viewModel.selectedBookProperty)
+                style {
+                    alignment = Pos.CENTER
+                    accentColor = c(Colors["primary"])
+                }
                 cellCache {
                     vbox(10) {
                         alignment = Pos.CENTER
-                        //label(it.book.title)
-                        button()
+                        label(it.titleKey)
+                        label(it.labelKey)
+                        button(messages["select"])
                     }
+                }
             }
+//
+    override fun onSave() {
+        runAsync{
+            viewModel.createProject()
```

Comment:
Might be able to put createProject call in a Completable instead of using `runAsync`, just to keep our approach to threading the same across the app

Reply:
i can just remove that. i don't believe i need that async